### PR TITLE
bubble sort

### PR DESCRIPTION
**Pull request by prernapriyali**
bubble sort using ruby

### CODE REVIEW

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Nice work so far. 👍 

- Good job implementing the `bubble_sort` method.

## However, there are some changes required 👀 🧰 🔨 🔧 

- For this project, it's also necessary to implement a `bubble_sort_by` method, as specified in The Odin Project:

image

- Also, please make sure to provide a more descriptive title for this Pull Request, as well as a summary of the things involved in creating this project. Perhaps a list of the new programming concepts you learned and applied.

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes  ♲

Hi @prernapriyali, I am [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  👏 You made good progress on this project. ✨ I applaud you for your hard work in implementing custom enumerable methods. Most requirements have done 👌, Kindly fix the following issues.

- [ ] Fix rubocop linting error
- [ ] bubble_sort_by is not working as expected.
For example:
` %w[hi hello h hey] #...` should return `[ "h", "hi", "hey", "hello"]`, but currently it returns `["hi", "h", "hey", "hello"]`

Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on implementing all requirements and getting this project approved. :+1:

You can merge this branch into the main branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime**

**Pull request merged**
